repose-aggregator/core/repose-core/src/main/scala/org/openrepose/powerfilter/ReposeFilter.scala

```diff
@@ -19,7 +19,193 @@
+                TraceIdLogger.trace("Adding plain text trans id to request: {}", traceGUID)
+                wrappedRequest.replaceHeader(REQUEST_ID, traceGUID)
+              }
+              val tracingHeader = wrappedRequest.getHeader(TRACE_GUID)
+              TraceIdLogger.info("Tracing header: {}", TracingHeaderHelper.decode(tracingHeader))
+              wrappedResponse.addHeader(TRACE_GUID, tracingHeader)
+            }
+            new ReposeFilterChain(
+              filterContextListToo.filterContexts,
+              chain,
+              None,
+              optMetricRegistry,
+              tracer
+            ).doFilter(wrappedRequest, wrappedResponse)
+          }) match {
+            case Success(_) =>
+              ThisLogger.trace("{} -- Successfully processed request", nodeId)
+            case Failure(use: URISyntaxException) =>
+              ThisLogger.debug(s"$nodeId -- Invalid URI requested: ${wrappedRequest.getRequestURI}", use)
+              wrappedResponse.sendError(SC_BAD_REQUEST, "Error processing request")
+            case Failure(e: Exception) =>
+              ThisLogger.error(s"$nodeId -- Issue encountered while processing filter chain.", e)
+              wrappedResponse.sendError(SC_BAD_GATEWAY, "Error processing request")
+            case Failure(t: Throwable) =>
+              ThisLogger.error(s"$nodeId -- Error encountered while processing filter chain.", t)
+              wrappedResponse.sendError(SC_BAD_GATEWAY, "Error processing request")
+              throw t
+          }
+        } finally {
+          closeSpan(wrappedResponse, scope)
+          if (!wrappedResponse.isCommitted) {
+            responseHeaderService.setVia(wrappedRequest, wrappedResponse)
+          }
+          wrappedResponse.commitToResponse()
+          optMetricRegistry.foreach { (mr: MetricRegistry) =>
+            markResponseCodeHelper(
+              mr,
+              wrappedResponse.asInstanceOf[HttpServletResponse].getStatus,
+              System.currentTimeMillis - startTime)
+          }
+        }
+        // Clear out the logger context now that we are done with this request
+        MDC.clear()
+        ThisLogger.trace("ReposeFilter returning response...")
+      case None =>
+        ThisLogger.error("ReposeFilter has not yet initialized...")
+        response.asInstanceOf[HttpServletResponse].sendError(SC_INTERNAL_SERVER_ERROR, "ReposeFilter not initialized")
+    }
+  }
+
+  override def destroy(): Unit = {
+    ThisLogger.trace("{} -- destroying ...", nodeId)
+    ThisLogger.info("{} -- Destroyed ReposeFilter", nodeId)
+  }
+
+}
+
+object ReposeFilter {
+  // NOTE: This is to workaround the Spring DelegatingFilterProxy ancestry which provides an Apache Log named logger.
+  // NOTE: This provided logger in turn conflicts with Scala's StrictLogging provided logger.
+  final val ThisLogger = LoggerFactory.getLogger(classOf[ReposeFilter].getName)
```

Comment:
Removal of WAR support solves this right? If so dump the war code now, they will never have to coexist.

Reply:
It sure did.